server/src/omtDocumentInformationProvider.ts

```diff
@@ -0,0 +1,359 @@
+     * Returns undefined when unable to resolve the link with the specified data.
+     * @param data the data from a document link
+     */
+    resolveLink(data?: DeclaredImportLinkData | unknown) {
+        if (data && isDeclaredImportLinkData(data)) {
+            return this.workspaceLookup.getModulePath(data.declaredImport.module);
+        }
+    }
+
+    /**
+     * Find all tsconfig files that apply to a given document
+     * and compile a list of paths annotations defined in those configurations.
+     * @param document the document that needs context.
+     */
+    private contextPaths(document: TextDocument) {
+        return new Map<string, string>(this.tsConfigFiles
+            // we need to filter for same parent because another config may have some of the same paths
+            .filter(uri => document.uri.startsWith(uri.substr(0, uri.lastIndexOf('/'))))
+            .reduce((paths: [string, string][], uri: string) => {
+                try {
+                    const file = readFileSync(uri, 'utf8');
+                    const json = JSON.parse(file);
+                    if (json.compilerOptions && json.compilerOptions.paths) {
+                        // now make a path from the config folder to the keys value and map that
+                        for (const key in json.compilerOptions.paths) {
+                            const relPath = json.compilerOptions.paths[key].toString();
+                            const newPath = resolve(dirname(uri), relPath);
+                            paths.push([key.substr(0, key.lastIndexOf('/*')), newPath]);
+                        }
+                    }
+                } catch (e) {
+                    // need to catch this error for when there is a JSON parsing error for example
+                    // we can still continue with other files
+                    console.error(e);
+                }
+                return paths;
+            }, []));
+    }
+
+    /**
+     * Scan the document for imports/declarations/used objects and returns an OmtDocumentInformation object.
+     * @param document the OMT file we are scanning
+     * @param resolveShorthand function to resolve shorthand annotations at the start of import paths
+     */
+    private getOmtDocumentInformation(document: TextDocument, shorthands: Map<string, string>): OmtDocumentInformation {
+        console.time('getOmtDocumentInformation for ' + document.uri);
+        const documentLinks: DocumentLink[] = [];
+        const calledObjects: OmtLocalObject[] = [];
+        // check all lines between 'import:' and another '(\w+):' (without any preceding spaces)
+        let isScanning = false;
+        const documentText = document.getText();
+        let fileImportsResult = undefined;
+        try {
+            fileImportsResult = getAvailableObjectsFromDocument(document, shorthands);
+        }
+        catch (error) {
+            if (error instanceof YAMLError) {
+                console.log(error);
```

Comment:
I think it was because of a test that uses the textFixture `imports.omt`. There are multiple YAML errors in that file, too many spaces online 4, and the url at line 6 should be encapsulated with quotes for example. 

If there are errors like that (and OMT can handle it) I think it is better to skip the YAML validation and ignore the YAMLError.